_assets/js/lunr.js

```diff
@@ -1728,7 +1728,7 @@
 
   /**
    * lunr.trimmer is a pipeline function for trimming non word
-   * characters from the begining and end of tokens before they
+   * characters from the beginning and end of tokens before they
```

Comment:
This is a vendored file. It should not be modified.

Reply:
Modification backed out. Sorry about that.